hook/privilege.go

```diff
@@ -0,0 +1,23 @@
+package hook
+
+import (
+	"fmt"
+
+	"github.com/Devatoria/go-mesos-executor/container"
```

Comment:
Please jump a line after this one to separate project packages from externals

Reply:
done